gridData/core.py

```diff
@@ -52,74 +52,140 @@ def _grid(x):
 
 
 class Grid(object):
-    """Class to manage a multidimensional grid object.
+    """A multidimensional grid object with origin and grid spacings.
 
-    The export(format='dx') method always exports a 3D object, the
-    rest should work for an array of any dimension.
+    :class:`Grid` objects can be used in arithmetical calculations
+    just like numpy arrays *if* they are compatible, i.e., they have
+    the same shapes and lengths. In order to make arrays compatible,
+    they an be resampled (:meth:`resample`) on a common grid.
 
-    The grid (Grid.grid) can be manipulated as a standard numpy
-    array.
+    The attribute :attr:`grid` that holds the data is a standard numpy
+    array and so the data can be directly manipulated.
 
-    The attribute Grid.metadata holds a user-defined dictionary that
-    can be used to annotate the data. It is saved with save().
-    """
-    default_format = 'DX'
+    Data can be read from a number of molecular volume/density formats
+    and written out in different formats with :meth:`export`.
 
-    def __init__(self, grid=None, edges=None, origin=None, delta=None,
-                 metadata=None, interpolation_spline_order=3,
-                 file_format=None):
-        """
-        Create a Grid object from data.
 
-        From a numpy.histogramdd()::
+    Parameters
+    ----------
+    grid : numpy.ndarray or str (optional)
+      Build the grid either from a histogram or density (a numpy nD
+      array) or read data from a filename.
 
-          grid,edges = numpy.histogramdd(...)
-          g = Grid(grid,edges=edges)
+    edges : list (optional)
+      list of arrays, the lower and upper bin edges along the axes
+      (such as the output by :func:`numpy.histogramdd`)
```

Comment:
```suggestion
      (same as the output by :func:`numpy.histogramdd`)
```
such -> same (might just be me but seems easier to understand)

Reply:
yes, oops, changed